Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+        private readonly string _currency = Settings.currencySymbol;
+        private readonly Account[] _accounts;
+
+        // this is used in conjuncture with FormatAndCommas
+        private const int _arrayBreak = 2;
+
+        public BalanceStats(string[] peopleAndBalances)
+        {
+            _accounts = AccountOps.BuildAccountList(peopleAndBalances);
+        }
+
+        /// <summary>
+        /// Return name and balance(current) of person who had the biggest historic balance.
+        /// </summary>
+        public string FindHighestBalanceEver()
+        {
+            if (_accounts is null)
+            {
+                return Messages.InvalidMessage;
+            }
+
+            Account highestBalanceAccount;
+            try
```

Comment:
Don't use try catch. First check the array size and handle that accordingly.

Reply:
I fixed it in [d1518fc](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/192/commits/d1518fc7884307a9ddc2f6074fa792682af6878d)